plugins/@grouparoo/hubspot/src/lib/export/options.ts

```diff
@@ -63,10 +63,20 @@ class OptionsHandler {
       primaryKey: { type: "pending", options: [] },
     };
     const objects = this.customObjects;
+    const objectsToSort = [];
     objects.map((object) => {
-      out.schemaId.options.push(object.objectTypeId);
-      out.schemaId.descriptions.push(object.name);
+      objectsToSort.push({
+        option: object.objectTypeId,
+        description: object.name,
+      });
     });
+    objectsToSort
+      .sort((a, b) => (a.description > b.description ? 1 : -1))
```

Comment:
Here (and in the other one), it would be better to do 
`.sort((a, b) => a.description.localeCompare(b.description))`
This handles the `equal -> 0` case